tox.ini

```diff
@@ -1,11 +1,55 @@
 [tox]
-skipsdist = True
+envlist=code-check
+skipsdist=True
 
-[testenv]
-setenv = PYTHONPATH = {toxinidir}
-deps =
+[testenv:code-check]
+basepython = python3
+recreate=True
+setenv =
+    PYTHONPATH = {toxinidir}
+deps=
     pre-commit
```

Comment:
do we need a `gitlint` as well?

Reply:
Pre-commit etc. are now handled via webhook (Meni's) we no longer need it here.